crates/meroctl/src/cli/bootstrap.rs

```diff
@@ -0,0 +1,41 @@
+use clap::{Parser, Subcommand};
+use const_format::concatcp;
+use eyre::Result as EyreResult;
+use start::StartBootstrapCommand;
+
+use super::Environment;
+
+mod start;
+
+pub const EXAMPLES: &str = r"
+  # Setup and run 2 nodes with demo app
+  $ meroctl -- --node-name node1 bootstrap start --merod-path /path/to/merod
```

Comment:
Why do we only need to provide name for node1?

Reply:
Only node_name does not have default value